src/bin/casc/args.rs

```diff
@@ -8,4 +8,11 @@ pub struct Args {
     pub input_file: Vec<String>,
     #[clap(default_value = "out.cil", short)]
     pub out_filename: String,
+    #[clap(
+        short,
+        multiple_values = true,
+        default_missing_value("all"),
```

Comment:
So this would imply that I can't declare a system named "all".

I'm fine with that, and actually we're not really good at enforcing that things aren't named reserved keywords yet.

Can you add a comment with this assumption?  I'll file an issue on github to add validation that declarations in general don't include reserved keywords.